recipes/menpowidgets/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "menpowidgets" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 84f80632ea4f9d4e0e710060724520146ce4d39b8adcac96e0d9c41fa4084d16
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py<35]
```

Comment:
Is there a reason this package cannot be noarch besides missing dependencies?

Our policy is that packages that can be noarch should be noarch even if they are missing dependencies. That way, if dependencies become available in the future, it doesn't requires rebuilding this package.